pom.xml

```diff
@@ -165,6 +173,16 @@
             <timeoutSeconds>30</timeoutSeconds>
             <timeoutRetries>2</timeoutRetries>
           </systemProperties>
+          <includes>
+            <include>**/Test*.java</include>
+            <include>**/*Test.java</include>
+            <include>**/*Tests.java</include>
+            <include>**/*TestCase.java</include>
+            <include>AnnotatedBaseTestClass.java</include>
+            <include>NeverEndingUdf.java</include>
+            <include>SlowlyFailingUdf.java</include>
```

Comment:
Nor are the others. I think you should be able to get rid of this whole `includes` section now that the tests are all renamed to end with `Test` and they will be detected by the default Surefire configuration.